R/tm_missing_data.R

```diff
@@ -141,15 +141,16 @@ ui_page_missing_data <- function(id, datasets, pre_output = NULL, post_output =
   )
 }
 
-srv_page_missing_data <- function(id, datasets, reporter, plot_height, plot_width, ggplot2_args) {
+srv_page_missing_data <- function(id, data, reporter, filter_panel_api, plot_height, plot_width, ggplot2_args) {
```

Comment:
Clicking SRC whilst on the By Variable Levels tab crashes the roxygen example app

![image](https://user-images.githubusercontent.com/15201933/187651083-9589cae9-047f-4571-b0a8-704d7abd1ff4.png)

Reply:
I'm on it ⏫

Reply:
solved:)